rocAL/include/pipeline/master_graph.h

```diff
@@ -189,7 +194,7 @@ class MasterGraph {
     bool _first_run = true;
     bool _processing;                                                             //!< Indicates if internal processing thread should keep processing or not
     const static unsigned SAMPLE_SIZE = sizeof(unsigned char);
-    int _remaining_count;                                                         //!< Keeps the count of remaining tensors yet to be processed for the user,
+    int _remaining_count = INT_MAX;                                                         //!< Keeps the count of remaining tensors yet to be processed for the user,
```

Comment:
Use minimum value here?

Reply:
It should be minimum value here i think

Reply:
It is INT_MAX only since for remaining count we need the minimum of different loaders, And need std::min for comparison